test/controllers/projects_controller_test.rb

```diff
@@ -337,6 +491,7 @@ def test_bad_location
   end
 
   def test_project_save_fail
```

Comment:
@mo-nathan
I'd like to delete this test. 
- It fails as shown below, and I have no idea how to fix it.
- I don't understand its internals and purpose, other than to prove that if it isn't saved, it can't be found.
- Deleting it won't remove any line of coverage.

Reply:
Actually deleting it causes line 345 in projects_controller.rb to not be covered.  I was essentially trying to get mocking working reasonable and it looks like I went down the wrong rabbit hole.  I think I have a better approach that I need to test some more and then I'll share it with you.